tests/bluetooth/common/testlib/include/testlib/scan.h

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_TESTS_BLUETOOTH_COMMON_TESTLIB_INCLUDE_TESTLIB_SCAN_H_
+#define ZEPHYR_TESTS_BLUETOOTH_COMMON_TESTLIB_INCLUDE_TESTLIB_SCAN_H_
+
+#include <zephyr/bluetooth/bluetooth.h>
+
+int bt_testlib_scan_find_name(bt_addr_le_t *result, char name[]);
```

Comment:
Since it doesn't provide any documentation, I'll have to guess based on name: Is the result of this scan stored in `result`? 

If so, then I suggest that we put the argument (`name`) before the result:
```suggestion
int bt_testlib_scan_find_name(const char name[], bt_addr_le_t *result);
```